serverless.yml

```diff
@@ -9,6 +9,8 @@ provider:
   memorySize: 256
   environment: ${file(serverless/environment.js)}
   $<<: ${file(serverless/provider.js)}
+  apiKeys:
+    - ${opt:stage}-siebel-api-key
```

Comment:
Seems we need to add some permissions to the jenkins instance role for this:

```

  Serverless Error ---------------------------------------
 
  User: arn:aws:sts::056154071827:assumed-role/jenkinsInstanceRole/i-0b5188e0d8b738b18 is not authorized to perform: apigateway:GET on resource: arn:aws:apigateway:us-east-1::/apikeys
 
  Stack Trace --------------------------------------------
```
from https://jenkins-prod.cru.org/job/scale-of-belief-import/job/master/3/console